scripts/utils/verify_scripts.js

```diff
@@ -110,19 +110,19 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
   ) {
     const log = logActivated ? (...a) => console.log(...a) : () => {}
 
-    const BatchExchangeArtifact = require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange")
-    const networkId = await web3.eth.net.getId()
-    const BatchExchange = new web3.eth.Contract(BatchExchangeArtifact.abi, BatchExchangeArtifact.networks[networkId].address)
-
-    const auctionElementsDecoded = await getOrdersPaginated(BatchExchange, pageSize)
     const bracketTraderAddresses = brackets.map((address) => address.toLowerCase())
-
-    // Fetch all token infos(decimals, symbols etc) and prices upfront for the following verification
-    const relevantOrders = auctionElementsDecoded.filter((order) => bracketTraderAddresses.includes(order.user.toLowerCase()))
     const BatchExchangeContract = Contract(require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange"))
     BatchExchangeContract.setProvider(web3.currentProvider)
     const exchange = await BatchExchangeContract.deployed()
 
+    // Fetch all token infos(decimals, symbols etc) and prices upfront for the following verification
+    const ordersObjects = await Promise.all(
+      bracketTraderAddresses.map(async (bracketAddress) =>
+        exchangeUtils.decodeOrders(await exchange.getEncodedUserOrders.call(bracketAddress))
+      )
+    )
+    const relevantOrders = [].concat(...ordersObjects)
```

Comment:
Like here, we could use
```
const relevantOrders = [].concat(...ordersObjects).filter(x => !!x)
```

Reply:
I'll give it a try right here.
```suggestion
    const relevantOrders = [].concat(...ordersObjects).filter(x => !!x)
```